### DISCUSSION TITLE
Equipo 5 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#11] [Proyecto Final]
Hola Carla
Aquí está el hilo en el foro para tu equipo. Podrías conversar aquí sobre tus avances con el proyecto y avisar el Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengas al respecto.
Con la emoción de seguir el desarrollo de tu proyecto,
Kasturi

**Comment by 0Wenn0** [comment]
Hola Carla, soy Wendy, moderadora del equipo de educación de Patrones hermosos. Estoy aquí por si necesitas apoyo o si tienes alguna duda respecto a tu proyecto final ¡Aún estás a tiempo! No desistas. Estoy aquí para ayudarte. No dudes en escribir por si tienes alguna duda o comentario.

**Comment by KasturiSShah** [comment]
Hola Carla,

Espero estés súper bien.

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar tu presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si te conviene la asignación de tu equipo a una de las dos sesiones de clausura del curso, o si quieres cambiar su día asignado al otro día.

Estamos (@0Wenn0 y yo) aquí para apoyarte. Cualquier pregunta sobre tu proyecto, déjanos saber por medio de este hilo y te contestaríamos pronto.

Saludos,
Kasturi
